builder-support/dockerfiles/Dockerfile.rpmbuild

```diff
@@ -21,14 +16,10 @@ RUN cd /pdns/builder-support/helpers/ && ./install_rust.sh
 @ENDIF
 
 @IF [ -n "$M_dnsdist$M_all" ]
-# We do not build Quiche (DNS over QUIC support) on el-7 because the clang
-# version is too old to build the 'boring-sys' crate needed by Quiche
-RUN if ! $(grep -q 'release 7' /etc/redhat-release); then \
-      cd /pdns/builder-support/helpers/ && ./install_rust.sh; \
-      yum install -y git cmake clang; \
-      cd /pdns/builder-support/helpers/ && \
-      ./install_quiche.sh; \
-    fi
+RUN cd /pdns/builder-support/helpers/ && ./install_rust.sh; \
+    yum install -y git cmake clang; \
```

Comment:
ISTM all these commands want && chaining for resilience, but this is not a new problem

Reply:
agreed. Would make sense as a separate commit if we're touching all this anyway.